test/fixtures/Destructuring_with_es2015/expected.js

```diff
@@ -0,0 +1,13 @@
+function _classCallCheck(instance, Constructor) { if (!(instance instanceof Constructor)) { throw new TypeError("Cannot call a class as a function"); } }
+
+var Foo =
+/**
+ * @param {string} x
+ */
+function Foo(_ref) {
```

Comment:
Interesting. The asserts are missing completely here. That's different than what I saw, but perhaps related?